README.md

```diff
@@ -355,25 +350,25 @@ In the whole examined sample, it is 331:479 of incorrect:correct (sum 3*270=810)
 ## Conclusions
```

Comment:
The proportions above could also be expressed as percentages (% correct of total entries), for me that would be quicker to understand